x-pack/plugins/enterprise_search/public/applications/enterprise_search_content/components/search_index/pipelines/types.ts

```diff
@@ -0,0 +1,13 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+export interface IInferencePipeline {
```

Comment:
Can we remove the first `I` for purposes of consistency? It doesn't look like we use that convention anywhere else in the enterprise search Kibana code.

Reply:
But let's merge this first :)